### PR TITLE
Add location weather article

### PR DESCRIPTION
**Pull request by Keats**
Update plugins to have ipython3 rendering
Add new requirements needed for ipython3
Ensure images do not overflow in articles

### CODE REVIEW

**Comment by Keats** [comment]
[rendered markdown](https://github.com/WeAreWizards/blog/blob/weather/content/articles/comparing-weathers.md)
[rendered ipynb](https://github.com/WeAreWizards/blog/blob/weather/content/notebooks/weather.ipynb)

**Comment by teh** [comment]
Can't comment on notebook, so adding notes here:

1. check for French punctuation !
2. Correct capitalization of brands (e.g. Google not google) and terms (CSV not csv), Canadians not canadians, May and June, not may and june, etc, British not british :)
3. When using terms that are python classes put them in backticks. `DataFrame` not dataframe
4. same for code `.describe()` not describe

> When people ask how were each of these places, the first thing that comes to my mind is the weather

When people ask me about these places the first thing ...

>  about the exact numbers and what

 about the exact numbers. And what ..

> First let's define the four locations we are going to look at:

We're going to look at the following four places (you aren't defining anything here)

>  depends on requests and BeautifulSoup4. 

maybe add links for requests and BS4

> below 10 

add degree on all temperaturs, i.e. 10°

> and air con is a necessity

missing full stop at end: it's a full sentence

File: content/articles/little-printer.md [hunk 1]
```diff
@@ -12,7 +12,6 @@ CEO Matt Berg tweeted this last year:
 <blockquote class="twitter-tweet" lang="en"><p lang="en" dir="ltr">I&#39;m after a London Python freelancer for a 2mo gig, Flask/tests/Redis/Bootstrap. Mail me... matt at interconnected dot org. Tell yr friends!</p>&mdash; Matt Webb (@genmon) <a href="https://twitter.com/genmon/status/537304016993386497">November 25, 2014</a></blockquote> <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
 
 I was hooked.
-
```
Comment (Gautier) [inline content/articles/little-printer.md]: I think little printer is already published so we should not change it to avoid messing up RSS readers

Comment (Keats) [inline content/articles/little-printer.md]: it would mess up if you change the date no? rss reader wouldn't notice a diff (afaik)

**Comment by Gautier** [comment]
I can't comment on the notebook directly neither. 
I mostly have comments on grammar and spelling as the content is a nice example of using a notebook.

The only comment not on grammar or spelling I have is that the end is a bit abrupt. Maybe add something like "I have shown how to use notebook to explore some data for a problem I care about, now go on and make your own comparisons!".

The following comments are to be added to Tom's list:

> # Alternative to using iterator

# Alternative to using a slice

I don't get the meaning of "# Why yes I did 40 degrees just to be able to fit the legend properly"

All graphs have their right sides cut out in the github rendered version, not sure about the pelican output.

space missing in "start = datetime.date(2014, 1,1)"

> # Let's stop things related to temperatures (humidity is not 
> # a temperature obviously but influences how that temperature feels

I think it's missing a parenthesis

>THAT bad and canadians know 

THAT bad and Canadians know 

> sometimes gets below 10 but never reach 0 while being dry

sometimes gets below 10 but never reaches 0 while being dry

> Okinawa looks really good until you see that summer humidity

Okinawa looks really good until you feel (or experience) that summer humidity

> air con is a necessity

I'd go for "Air conditioning". I could argue that "air con" is a bad (but common) abbreviation. "air-con" or "aircon" could work, but "Air conditioning" is better anyway in a written text.

>To finish that notebook

To finish this notebook

>We can see it's raining quite a bit in Okinawa since they have a rainy season (may-june) and a typhoon season (june-november).

We can see it's raining quite a bit in Okinawa since they have a rainy season (May-June) and a typhoon season (June-November).

> Nice is probably the nicest,

nice

> (along with salary, atmosphere, friends/family etc)

(along with salary, atmosphere, friends/family etc.)

> british weather

British(!) weather

>2 weeks of summers

2 weeks of summer (I think)

**Pull request merged**